apps/aecore/test/aec_tx_pool_tests.erl

```diff
@@ -79,7 +79,7 @@ tx_pool_test_() ->
                {GenesisBlock, _} = aec_block_genesis:genesis_block_with_state(),
                aec_test_utils:start_chain_db(),
                ok = aec_chain_state:insert_block(GenesisBlock),
-               {TopBlock, TopState} = aec_chain:top_block_with_state(),
+               {TopBlock, _TopState} = aec_chain:top_block_with_state(),
```

Comment:
Use `aec_chain:top_block/0` to not get the state since you don't need it anymore.